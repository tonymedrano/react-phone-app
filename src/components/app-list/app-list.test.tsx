import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import AppList from './app-list';

const { asFragment } = render(<AppList />);

test('<AppList />', () => {
    const firstRender = asFragment();
    expect(firstRender).toMatchSnapshot(asFragment());
});