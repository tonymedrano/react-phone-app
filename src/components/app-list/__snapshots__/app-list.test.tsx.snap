// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppList /> 1`] = `Object {}`;
